Decisions.Box/BoxSettings.cs

```diff
@@ -0,0 +1,394 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using Box.V2;
+using Box.V2.Auth;
+using Box.V2.Config;
+using Box.V2.JWTAuth;
+using DecisionsFramework;
+using DecisionsFramework.Data.DataTypes;
+using DecisionsFramework.Data.ORMapper;
+using DecisionsFramework.Design.ConfigurationStorage;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Properties;
+using DecisionsFramework.ServiceLayer;
+using DecisionsFramework.ServiceLayer.Actions;
+using DecisionsFramework.ServiceLayer.Actions.Common;
+using DecisionsFramework.ServiceLayer.Services.Accounts;
+using DecisionsFramework.ServiceLayer.Services.Accounts.ActiveDirectory;
+using DecisionsFramework.ServiceLayer.Services.Administration;
+using DecisionsFramework.ServiceLayer.Services.ExternalServiceReference;
+using DecisionsFramework.ServiceLayer.Services.Folder;
+using DecisionsFramework.ServiceLayer.Utilities;
+
+namespace Decisions.Box;
+
+[ORMEntity("box_settings")]
+[DataContract]
+[Writable]
+public class BoxSettings : AbstractModuleSettings, IInitializable, INotifyPropertyChanged, IValidationSource
+{
+    // Client ID
+     [ORMField]
+     private string clientId;
+
+     [DataMember]
+     [WritableValue]
+     [PropertyHiddenByValue(nameof(UseJson), true, true)]
```

Comment:
What is `UseJson`? What does it do?

Reply:
Could we rename this to `UseJsonForAuth` or put it under an `Auth Type` category or something?

Reply:
Added an Alternate Authentication category for these two checkboxes and better titles.